NEWS.md

```diff
@@ -3,6 +3,9 @@
 * Examples now use `scda.2022` instead of `scda.2021`.
 * Fixed crash in `tm_g_heat_bygrade` when not plotting `Conmed`. 
 * Added validation to `tm_g_spiderplot.R` that checks if there are duplicates in X and Y facet variables.
+* Improved input validation and used the `teal::validate_inputs` mechanism to send validation messages to the output panel.
+* Removed `purrr` from dependencies.
+* Removed redundant formal arguments from add added argument checks to `tm_g_patient_profile`.
```

Comment:
This needs rephrasing.

Also can we have a PR in teal.gallery (earlydev app) which removes these arguments otherwise that app is broken